test/features/pagecontent/pagecontent.js

```diff
@@ -342,6 +343,28 @@ describe('page content access', function() {
             assert.deepEqual(e.body.detail, 'title-invalid-characters');
         });
     });
+
+    it('should return the latest revision for flagged titles on no-cache', function() {
+        return preq.get({
+            uri: server.config.deLabsBucketURL + '/html/FRtest',
+            headers: { 'cache-control': 'no-cache' }
+        }).then(function(res) {
+            const retRev = mwUtil.parseETag(res.headers.etag).rev;
+            assert.deepEqual(res.status, 200);
+            assert.notDeepEqual(retRev, 20657);
+        });
+    });
+
+    it('should return the stable revision for flagged titles', function() {
+        return preq.get({
+            uri: server.config.deLabsBucketURL + '/html/FRtest'
+        }).then(function(res) {
+            const retRev = mwUtil.parseETag(res.headers.etag).rev;
+            assert.deepEqual(res.status, 200);
+            assert.deepEqual(retRev, 20657);
```

Comment:
Em why is the stable the same as the latest? Are we actually testing anything here? If they're the same the tests will pass even without the flagged rev filter.

Reply:
This tests asserts `deepEqual()`, the other does `notDeepEqual()` ;)

Reply:
Oh indeed. My bad, sorry.